reporter/html.js

```diff
@@ -754,6 +754,20 @@ QUnit.testDone(function( details ) {
 		time.innerHTML = details.runtime + " ms";
 		testItem.insertBefore( time, assertList );
 	}
+
+	// Only add this information if stackTrace is supported
```

Comment:
Replace this comment with something explaining why this is here, or what it does, something like:

```js
// Show the source of the test when showing assertions
```